docs/oras-auth-provider.md

```diff
@@ -0,0 +1,144 @@
+# ORAS Authentication Provider
+
+The ORAS referrer store handles all registry and referrer resolution operations. ORAS provides authentication credentials to access referrer artifacts. Ratify contains many Authentication Providers to support different authentication scenarios. The user specifies which authentication provider to use in the configuration.
+
+The `auth-provider` section of configuration file specifies the authentication provider. Ratify **requires** the `name` field to bind to the correct provider implementation. 
```

Comment:
suggestion: You can use Notational conventions to indicate the criticality https://github.com/opencontainers/runtime-spec/blob/main/spec.md#notational-conventions